.zenodo.json

```diff
@@ -20,6 +20,21 @@
       "name": "Halchenko, Yaroslav O.",
       "orcid": "0000-0003-3456-2493"
     },
+    {
+      "affiliation": "",
```

Comment:
Centre for Modern Interdisciplinary Technologies, Nicolaus Copernicus University in Toruń